### PR TITLE
fix(docs-infra): update examples to Angular 13.1

### LINKED ISSUE #44480: Incorrect typescript version in examples
### Description

Many of the current examples (I imagine that all the examples for Angular 13?) seem to have incorrect dependencies, and they fail to run in Stackblitz. See for instance https://github.com/stackblitz/webcontainer-core/issues/473 and all the linked issues.

### What is the affected URL?

https://angular.io/generated/live-examples/getting-started-v0/stackblitz.html

### Please provide the steps to reproduce the issue

* Go to URL
* Wait for automatic scripts in Stackblitz to run (`npm install && turbo start`)
* Observe following error:
```
node:6) UnhandledPromiseRejectionWarning: Error: The Angular Compiler requires TypeScript >=4.4.2 and <4.5.0 but 4.5.4 was found instead.
    at checkVersion (file:///home/projects/j2wdau--run/node_modules/@angular/compiler-cli/bundles/index.js:17524:11)
    at verifySupportedTypeScriptVersion (file:///home/projects/j2wdau--run/node_modules/@angular/compiler-cli/bundles/index.js:17528:3)
    at new NgtscProgram (file:///home/projects/j2wdau--run/node_modules/@angular/compiler-cli/bundles/index.js:17678:7)
    at AngularWebpackPlugin.updateAotProgram (/home/projects/j2wdau--run/node_modules/@ngtools/webpack/src/ivy/plugin.js:321:32)
    at eval (/home/projects/j2wdau--run/node_modules/@ngtools/webpack/src/ivy/plugin.js:198:24)
    at Hook.eval [as call] (eval at _0x5eef21 (https://j2wdau--run.w.staticblitz.com/blitz.731360fb19d99dfc563f19370095b1e3465a14ed.js:11:221412), <anonymous>:20:1)
    at Hook.CALL_DELEGATE [as _call] (/home/projects/j2wdau--run/node_modules/tapable/lib/Hook.js:14:14)
    at Compiler.newCompilation (/home/projects/j2wdau--run/node_modules/webpack/lib/Compiler.js:1054:30)
    at eval (/home/projects/j2wdau--run/node_modules/webpack/lib/Compiler.js:1099:29)
    at eval (eval at _0x5eef21 (https://j2wdau--run.w.staticblitz.com/blitz.731360fb19d99dfc563f19370095b1e3465a14ed.js:11:221412), <anonymous>:31:1)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:6) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
⠼ Generating browser application bundles (phase:
```

### Please provide the expected behavior vs the actual behavior you encountered

Compilation should run without issues.

### Please provide a screenshot if possible

screenshot

### Please provide the exception or error you saw

_No response_

### Is this a browser-specific issue? If so, please specify the device, browser, and version.

_No response_

### PR DESCRIPTION
**Pull request by crisbeto**
Updates AIO and the live examples to 13.1.0 in order to resolve a build error when opened on Stackblitz. They were using TS 4.5 with Angular ~13.0.0 which doesn't support 4.5.

Fixes #44480.

### CODE REVIEW

File: goldens/size-tracking/aio-payloads.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "aio": {
     "master": {
       "uncompressed": {
-        "runtime": 4436,
+        "runtime": 4330,
```
Comment (gkalpak) [inline goldens/size-tracking/aio-payloads.json:5]: :tada:

File: aio/package.json [hunk 2]
```diff
@@ -103,12 +103,12 @@
...
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "13.0.4",
+    "@angular-devkit/build-angular": "^13.1.0",
     "@angular-eslint/builder": "^13.0.0",
     "@angular-eslint/eslint-plugin": "^13.0.0",
     "@angular-eslint/eslint-plugin-template": "^13.0.0",
     "@angular-eslint/template-parser": "^13.0.0",
-    "@angular/cli": "13.0.4",
+    "@angular/cli": "^13.1.0",
```
Comment (gkalpak) [inline aio/package.json:111]: This one is unrelated to the docs examples, but happy to let it slip through 🙈 😃

**Review by gkalpak — APPROVED** [review]
Thx, @crisbeto ✨

**Review by jelbourn — APPROVED** [review]
LGTM

Reviewed-for: global-approvers

**Comment by crisbeto** [comment]
Merge assistance: there's global approval.

**Review by dylhunn — APPROVED** [review]

**Comment by alxhub** [comment]
This PR was merged into the repository by commit 7c246d513b233a2098f2e4a673ce989b76419b82.

**Pull request closed without merging**